packages/amplify-cli/src/extensions/amplify-helpers/triggerTypes.ts

```diff
@@ -0,0 +1,105 @@
+// export interface TriggerModules {
```

Comment:
leftover?

Reply:
I am keeping this to replace any type of trigger in auth. Will cover it when converting this full package to ts